vespa/application.py

```diff
@@ -210,14 +214,35 @@ def feed_data_point(self, schema: str, data_id: str, fields: Dict):
         response = self.http_session.post(end_point, json=vespa_format, cert=self.cert)
         return response
 
-    def feed_batch(self, batch):
+    def _feed_batch_sync(self, schema: str, batch: List[Dict]):
+        return [
+            self.feed_data_point(schema, data_point["id"], data_point["fields"])
+            for data_point in batch
+        ]
+
+    async def _feed_batch_async(self, schema: str, batch: List[Dict]):
+        async with VespaAsync(self) as async_app:
+            return await async_app.feed_batch(schema=schema, batch=batch)
+
+    def feed_batch(self, schema: str, batch: List[Dict], asynchronous=False):
```

Comment:
I think one usually wants to feed as fast as possible, so unless there are negative consequences, the `asynchronous` arguments should default to `True`. Or better, `synchronous=False`.

Reply:
The only reason that I set `asynchronous=False` is that `asynchronous=True` requires an extra step from the user when using it where there is already a running loop in place, such as in the Jupyter notebooks. In those cases, the user needs to add an await keyword for the coroutine to run. What do you think?